.github/workflows/test-template-changes.yml

```diff
@@ -41,11 +41,11 @@ jobs:
       run: ./src/tools/scripts/test_template_changes.sh
     - name: 'Comment PR'
       uses: actions/github-script@0.3.0
-      if: github.event_name == 'pull_request' && env.PR_COMMENT != ''
+      if: github.event_name == 'pull_request' && steps.test-template-changes.outputs.PR_COMMENT != ''
```

Comment:
I wonder, if object attribute names with hyphens (such as `steps.test-template-changes`) are accessible with dot-notation? In some situations such attributes with hyphens or spaces are accessed using square bracket notation. Unless I am absolutely sure, I would change them to the corresponding underscore-form.

Reply:
Tested this and no issue. I could see where having dots in the name might cause issues but can't see why dashes would.

Anyway discovered other issues when testing so stopped using output variables and went back to env variables so this is a moot point.

Reply:
> I could see where having dots in the name might cause issues but can't see why dashes would.

Well, there are situations where a dash will be considered as a separate token and be treated as minus sign. Try the following in a web browser's DevTools console:

```
o = {"a": "b", "c-d": "e"}
>> {a: "b", c-d: "e"}

o["a"]
>> "b"

o.a
>> "b"

o["c-d"]
>> "e"

o.c-d
>> Uncaught ReferenceError: d is not defined
    at <anonymous>:1:5
```

Reply:
Interesting. Still doesn't matter as not using this code but removed the id anyway in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1367/commits/d9f104ed6eeddce0a1585dea4343b715782c1437 as it wasn't being used.